app/src/main/java/com/danvelazco/fbwrapper/activity/BaseFacebookWebViewActivity.java

```diff
@@ -537,6 +559,10 @@ public void onProgressChanged(WebView view, int progress) {
         // Hide the progress bar as soon as it goes over 85%
         if (progress >= 85) {
             mProgressBar.setVisibility(View.GONE);
+
+            //Checks if the swipe refresh indicator is active. Turns off if it is
+            if(mSwipeRefreshLayout.isRefreshing())
+                mSwipeRefreshLayout.setRefreshing(false);
```

Comment:
I'm a bit of a newbie. I don't exactly understand by what you mean by wrapping with curly braces. Mind explaining?